app/models/driver_commission_history_adapter.rb

```diff
@@ -1,11 +1,13 @@
 # DriverCommissionHistoryAdapter
+# rubocop:disable Metrics/LineLength,
 class DriverCommissionHistoryAdapter < GrossmanAdapter
   QUERY_ALL = <<SQL
         SELECT
           DriverId
         , Backhauls
         , FreightRevenue
-        , CustomerId
+        , Customers.CustomerId
+        , Customers.Name
```

Comment:
Can you confirm that we can't alias these names, I think this would be better if we could return this as CustomerName.

Thanks.